### ISSUE TITLE
Conflict with SublimeJSCSFormatter

### ISSUE THREAD

**Issue by hudochenkov** [TheSavior/ESLint-Formatter#4]
I have both your ESLint-Formatter and SublimeJSCSFormatter installed. When I run ESLint Formatter I get this error:

```
Error: No configuration found. Add a .jscsrc file to your project root or use the -c option.
```

And this is correct, I have no `.jscsrc` for current project, but I have `.eslintrc.json` and run ESLint Formatter.

I think problem is in the name of main class `FormatJavascriptCommand `. Both plugins use this name for command and Sublime Text decides which plugin will run this command. I suggest to use different commands in this plugins.

**Comment by TheSavior** [comment]
Hmm. They also both use the same shortcut keys.

I'll change FormatJavascriptCommand in this plugin. Thanks for the issue

**Comment by hudochenkov** [comment]
Thank you!

**Comment by TheSavior** [comment]
Released as 1.0.1
